cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -454,7 +455,8 @@ struct ColumnChunkDesc {
 
   float list_bytes_per_row_est{};  // for LIST columns, an estimate on number of bytes per row
 
-  bool is_strings_to_cat{};  // convert strings to hashes
+  bool is_strings_to_cat{};    // convert strings to hashes
+  bool is_large_string_col{};  // `true` if string data exceeds 2GB limit
```

Comment:
Should we write this differently? It's really about the offset type, not the raw size, if I understand correctly.
```suggestion
  bool is_large_string_col{};  // `true` if string data uses 64-bit offsets
```